src/components/config-tabs/ConfigTabs.module.css

```diff
@@ -2,74 +2,17 @@
     margin: 20px auto;
 }
 
-.blocTabs {
-    display: flex;
-}
-.tabs {
-    padding: 10px 0;
-    text-align: center;
-    flex-grow: 1;
-    background: rgba(128, 128, 128, 0.075);
-    cursor: pointer;
-    border-bottom: 1px solid rgba(22, 132, 33, 0.274);
-    box-sizing: content-box;
-    position: relative;
-    outline: none;
-    border-radius: 2px;
-}
-.tabs:not(:last-child) {
-    border-right: 1px solid rgba(0, 0, 0, 0.274);
-}
-
-.activeTabs {
-    flex-grow: 1;
-    background: white;
-    border-bottom: 1px solid transparent;
-}
-
-.activeTabs::before {
-    content: '';
-    display: block;
-    position: absolute;
-    top: -5px;
-    left: 50%;
-    transform: translateX(-50%);
-    width: calc(100% + 2px);
-    height: 5px;
-    background: rgb(88, 147, 241);
+.subContainer {
+    width: 95%;
+    margin: auto;
 }
 
-button {
-    border: none;
-}
-.contentTabs {
-    flex-grow: 1;
-}
-.content {
-    background: white;
-    padding: 20px;
-    width: 100%;
-    height: 100%;
-    display: none;
-}
-.content h2 {
-    padding: 0px 0 5px 0px;
-}
-.content hr {
-    width: 100px;
-    height: 2px;
-    background: #222;
-    margin-bottom: 5px;
-    margin-left: 0;
-}
-.content p {
+.exitConfigurationsContainer {
+    height: 48px;
```

Comment:
I had to set the height here directly because `height: 100%` was not working. I guess because the parent TabBar div does not have a known height?